.trivyignore

```diff
@@ -0,0 +1,3 @@
+CVE-2022-40897
```

Comment:
Can we fix these?

Reply:
There's an item in the backlog to address linting properly. IMHO this is out of scope